src/coins.ts

```diff
@@ -29,8 +31,7 @@ export const coins = CoinMap.fromCoins([
   account('txlm', 'Testnet Stellar', Networks.test.stellar, 7, UnderlyingAsset.XLM),
   account('susd', 'Silvergate USD', Networks.main.susd, 2, UnderlyingAsset.USD),
   account('tsusd', 'Testnet Silvergate USD', Networks.test.susd, 2, UnderlyingAsset.USD),
-  utxo('zec', 'ZCash', Networks.main.zCash, UnderlyingAsset.ZEC),
-  utxo('tzec', 'Testnet ZCash', Networks.test.zCash, UnderlyingAsset.ZEC),
+  account('ofc', 'Offchain', Networks.test.ofc, 2, UnderlyingAsset.USD, OfcCoin.DEFAULT_FEATURES, '$', CoinKind.FIAT),
```

Comment:
This probably shouldn't be a thing. There is no ofc asset, there are only tokens. We use ofcusd to know the fiat balance.